test/fuse_failover/framework_test.go

```
@@ -0,0 +1,509 @@
+}
+
+func (c *failoverCluster) MountDir(i int) string { return c.mountPoints[i] }
+
+func (c *failoverCluster) Stop() {
+	if c == nil {
+		return
+	}
+	c.cleanupOnce.Do(func() {
+		for i := len(c.mountCmds) - 1; i >= 0; i-- {
+			c.stopCmd(c.mountCmds[i], syscall.SIGTERM)
+			exec.Command("fusermount3", "-u", c.mountPoints[i]).Run()
```

golangci-lint (2.12.2)

[error] Error return value of `(*os/exec.Cmd).Run` is not checked

(errcheck)